contracts/vendor/@openzeppelin/contracts/3.4.1/commit.info

```diff
@@ -0,0 +1 @@
+cec0800c541c809f883a37f2dfb91ec4c90263c5
```

Comment:
Wondering if we should just use submodules at this point? Do you see any benefit one way or the other?

I'm leaning towards submodules because this seems error prone and easy to miss when updating. Using a submodule would enforce it stays up to date and have good support for automating steps.

Reply:
Yeah, I'd prefer submodules rather than doing this manually.